docs/develop/module-specifications/spec-governance.mdx

```diff
@@ -301,32 +330,33 @@ The minimum deposit amount for a proposal to enter a voting period. Currently 51
 - type: `time.Duration` (seconds)
 - `"max_deposit_period": "604800s"`
 
-Maximum period for Luna holders to deposit on a proposal. Currently 1 week.
+The maximum period of time for a proposal to meet the minimum deposit. Currently 1 week.
 
 ### Quorum
 
 - type: `Dec`
 - `"quorum": "0.100000000000000000",`
 
-Minimum percentage of total stake needed to vote for a result to be considered valid.
+The minimum number of votes needed for a proposal to be valid. Currently, 10% of all staked Luna must vote to reach `Quorum`. 
 
 ### Threshold
 
 - type: `Dec`
 - default value: 50%
 
-Minimum proportion of Yes votes for a proposal to pass.
+The minimum number of `Yes` votes needed for a proposal to pass. Currently, 50% of all votes must be `Yes`. `Abstain` votes are excluded from this tally. 
 
 ### Veto
 
 - type: `Dec`
-- default value: `0.33`
+- default value: `0.334`
 
-Minimum value of Veto votes to Total votes ratio for proposal to be vetoed.
+The minimum value of `NoWithVeto` votes needed for a proposal to be vetoed. If 33.4% or more of the total votes cast are `NoWithVeto`, the proposal fails, and the creator's deposit is not returned. 
```

Comment:
comma not needed after "the proposal fails"

Reply:
"the creator's deposit is not returned" is an independent clause, so a comma is needed.